srcpkgs/libqb/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libqb'
+pkgname=libqb
+version=1.0.3
+revision=1
+hostmakedepends="glib-devel pkg-config"
+short_desc="libqb is a library with the primary purpose of providing high-performance, reusable features for client-server architecture, such as logging, tracing, inter-process communication (IPC), and polling."
+only_for_archs="x86_64"
+build_style=gnu-configure
+maintainer="Kevin Klopfenstein <kk@sudo-i.net>"
+license="GPL-3.0-or-later"
```

Comment:
Wrong license, please check the appropriate license for the software you're packaging this is very important.